pyproject.toml

```diff
@@ -95,6 +95,9 @@ include = [
     "/docs/sphinx",
 ]
 
+[tool.hatch.build.targets.wheel]
+packages = ["elasticsearch_serverless"]
+
```

Comment:
This is required by a recent Hatch change: https://github.com/pypa/hatch/issues/1113